docs/basics/async.mdx

```diff
@@ -80,6 +110,57 @@ const Component = () => {
 
 > There's no way to know as of now if an atom suspends because of `read` or `write`.
```

Comment:
We can remove this. It's a note for pre-1.4. `write` will not suspend. Exception: if you `write` a promise object, it will suspend, but "you know it".

Reply:
@dai-shi That part was so misleading me that I didn't even dare to touch it.
So you confirm me that:
```js

cont [_, setNum] = useAtom(writeAsync) is never gonna suspend
```

only the below will
```js
cont [_, setNum] = useAtom(writeAsync) is never gonna suspend

React.useEffect(() => {
   setTimeout(() => { setNum() }, 1000) // This triggers the suspend
}, [])
```

Are we good? 

> if you write a promise object, it will suspend, but "you know it".

I'm not sure I understood that part

> How about moving "Async" section to the first item of "GUIDES"?

Arf I feel bad about it. I would have really liked for people to consider it basics. But I feel your pain of attacking utils straight in the basics... In that case, I would propose a re-order of the menu to put Async in top of the scene.

What do you think?
Just a proposition, we can fallback to just moving Async if you want, for now ;)

```
BASICS
- Concepts
- Primitives

API
- Core
- Utils

GUIDES
- Async
- Resettable <== Too simply to be a guide, should only be an util/recipe, no?
- Persistence
- Atoms in atom
- Composing atoms

EXPERIENCE
- TypeScript
- Debugging
- Devtools
- Babel
- Testing
- Core Internals

BRIDGES
- Next.js
- React Native
- Vite

SHOWCASE:
- Comparison
- Showcase
```

Reply:
@dai-shi I'm not sure how to document the loadable util correctly, tho.
Is there anything more that I should be aware of, more of the simple usage:

```jsx
const loadableAtom = loadable(asyncAtom)
// Does not need to be wrapped by a <React.Suspense> element
const Component = () => {
  const value = useAtom(loadableAtom)
  if (value.error) return <Text>Error</Text>
  if (value.state === 'pending') {
    // 'pending' | 'hasData'
    return <Text>Loading...</Text>
  }
  console.log(value.data) // Results of the Promise
  return <Text>Value: {value.data}</Text>
}
```

If it's good, should I add it as the last util ? #20 ?

Reply:
```js
cont [_, setNum] = useAtom(writeAsync) is never gonna suspend

React.useEffect(() => {
   setTimeout(() => { setNum() }, 1000) // This triggers the suspend
}, [])
```
It will not trigger suspend.